apps/react-workshop/src/InformationPanel.stories.tsx

```diff
@@ -39,7 +39,7 @@ export const Basic = () => {
       },
     ],
     [],
-  );
+  ) satisfies Column<{ name: string }>[];
```

Comment:
Does `satisfies Column[]` not work? because this somewhat defeats the purpose of using `satisfies`